src/components/Coldcard/ColdcardFileReader.jsx

```diff
@@ -2,14 +2,9 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import Dropzone from "react-dropzone";
-import {
-  Box,
-  Button,
-  FormHelperText,
-  Grid,
-  TextField,
-} from "@material-ui/core";
-import { CloudUpload as UploadIcon } from "@material-ui/icons";
+import { Buffer } from "buffer/";
```

Comment:
Is this `Buffer` necessary? It doesn't look like it was here before and I don't see anything else changed in the file that would require it. If it is, should the trailing slash be there?

Reply:
It's being used on line 105 `const psbtHex = Buffer.from(psbtData).toString("hex");` so we should keep it right?

Reply:
Yep! Thanks. Didn't dig in to the file, just the diff. Thanks for confirming